docs/openstack-openedx/supported-deployment.md

```diff
@@ -0,0 +1,99 @@
+# OpenStack Production Deployment (Proposal)
+
+edX does not officially provide any supported method for deploying a production-grade instance of Open edX. However, the following *proposed* method for setting up a production-grade instance on OpenStack is available to the community, and support for this method will be provided by OpenCraft, if there is sufficient community interest in this approach. For context, this method of deployment is used in production for over 100 production instances of Open edX on [OpenCraft's hosting platform](http://opencraft.com/hosting/).
```

Comment:
Can we not promise the support for this? As mentioned in the past I'd rather under promise, and over deliver.